test/interop/python/noPicError/COMPOPTS

```diff
@@ -0,0 +1 @@
+--library --library-python --library-dir=lib
```

Comment:
Couldn't you put this test under `test/interop/python/errorMessages` as a subdirectory?

Reply:
Sure.